homeassistant/components/matrix/__init__.py

```diff
@@ -178,11 +186,13 @@ def __init__(
             homeserver=self._homeserver, user=self._mx_id, ssl=self._verify_tls
         )
 
-        self._listening_rooms = listening_rooms
+        self._listening_rooms: dict[RoomAnyID, RoomID] = {
+            room_alias_or_id: RoomID("") for room_alias_or_id in listening_rooms
+        }
```

Comment:
Is `RoomID("")` a valid room id, or would the client raise an error when it will be used? Example in `_join_room`

```suggestion
        self._listening_rooms: dict[RoomAnyID, RoomID] = {}
```

Why don't you initialize an empty dict?

Reply:
Done